### PR TITLE
Marked some strings translatable.

### PR DESCRIPTION
**Pull request by urbalazs**
Can you sync these changes to Transifex after commit?

### CODE REVIEW

File: Kernel/Config/Files/OTRSMasterSlave.xml [hunk 1]
```diff
@@ -270,7 +270,7 @@
         <Group>MasterSlave</Group>
         <SubGroup>Frontend::Agent::Ticket::ViewMasterSlave</SubGroup>
         <Setting>
-            <String Regex="">[% Translate("MasterSlave") | html %]!</String>
+            <String Regex=""></String>
         </Setting>
```
Comment (carlosfrodriguez) [inline Kernel/Config/Files/OTRSMasterSlave.xml]: This was intentionally removed?

File: Kernel/Modules/AdminDynamicFieldMasterSlave.pm [hunk 2]
```diff
@@ -58,19 +58,21 @@ sub Run {
         );
     }
     return $LayoutObject->ErrorScreen(
-        Message => "Undefined subaction.",
+        Message => $LayoutObject->{LanguageObject}->Translate('Undefined subaction.'),
     );
```
Comment (carlosfrodriguez) [inline Kernel/Modules/AdminDynamicFieldMasterSlave.pm]: ErrorScreen() uses Error() that calls Error.tt that already translate the message, a recommendation here is to import Translatable() from LayoutObject and mark the strings to translation, then they could be added to transifex.

This applies to several parts in this commit.

File: Kernel/Modules/AdminDynamicFieldMasterSlave.pm [hunk 3]
```diff
@@ -97,12 +99,13 @@ sub _AddAction {
...
     my $ParamObject = $Kernel::OM->Get('Kernel::System::Web::Request');
+    my $LayoutObject = $Kernel::OM->Get('Kernel::Output::HTML::Layout');
 
     for my $Needed (qw(Name Label FieldOrder)) {
         $GetParam{$Needed} = $ParamObject->GetParam( Param => $Needed );
         if ( !$GetParam{$Needed} ) {
             $Errors{ $Needed . 'ServerError' }        = 'ServerError';
-            $Errors{ $Needed . 'ServerErrorMessage' } = 'This field is required.';
+            $Errors{ $Needed . 'ServerErrorMessage' } = $LayoutObject->{LanguageObject}->Translate('This field is required.');
         }
```
Comment (carlosfrodriguez) [inline Kernel/Modules/AdminDynamicFieldMasterSlave.pm]: Server Error Messages are normally translated in the .tt files, a recommendation here is to import Translatable() from LayoutObject and mark the strings to translation, then they could be added to transifex.

This applies to several parts in this commit.

File: Kernel/Modules/AgentTicketMasterSlave.pm [hunk 4]
```diff
@@ -98,7 +98,7 @@ sub Run {
     # error screen, don't show ticket
     if ( !$Access ) {
         return $LayoutObject->NoPermission(
-            Message    => "You need $Config->{Permission} permissions!",
+            Message    => $LayoutObject->{LanguageObject}->Translate('You need %s permissions!', $Config->{Permission}),
             WithHeader => 'yes',
```
Comment (carlosfrodriguez) [inline Kernel/Modules/AgentTicketMasterSlave.pm]: NoPermission() uses NoPermission.tt that already translate the message, a recommendation here is to import Translatable() from LayoutObject and mark the strings to translation, then they could be added to transifex.

File: Kernel/Output/HTML/TicketBulk/MasterSlave.pm [hunk 5]
```diff
@@ -155,12 +156,12 @@ sub _GetMasterSlaveData {
...
-        Master => 'New Master Ticket',
+        Master => $LayoutObject->{LanguageObject}->Translate('New Master Ticket'),
     );
 
     if ($UnsetMasterSlave) {
-        $Data{UnsetMaster} = 'Unset Master Tickets';
-        $Data{UnsetSlave}  = 'Unset Slave Tickets';
+        $Data{UnsetMaster} = $LayoutObject->{LanguageObject}->Translate('Unset Master Tickets');
+        $Data{UnsetSlave}  = $LayoutObject->{LanguageObject}->Translate('Unset Slave Tickets');
     }
```
Comment (carlosfrodriguez) [inline Kernel/Output/HTML/TicketBulk/MasterSlave.pm]: Default MasterSlave dynamic field configuration is set to translate the values, so a translation here is not needed, however if we mark the strings as translatable , they could be added to transifex.

**Comment by carlosfrodriguez** [comment]
Hi @urbalazs, 

Thank you for your pull request I add some notes, where I believe a translation is not really needed, other places might require to mark the strings as translatable, please refer to ticket modules to get some examples of the usage of Translatable();

((enjoy))
cr

**Comment by urbalazs** [comment]
Hi @carlosfrodriguez 
I changed some "$LayoutObject->{LanguageObject}->Translate()" to "Translatable()".
In the file "Kernel/Modules/AgentTicketMasterSlave.pm" at line 101: the message has a parameter, therefore I used Translate() instead of Translatable().

File: Kernel/Modules/AgentTicketMasterSlave.pm [hunk 6]
```diff
@@ -23,6 +23,7 @@ use warnings;
```
Comment (carlosfrodriguez) [inline Kernel/Modules/AgentTicketMasterSlave.pm]: Unfortunately this file is outdated form the framework, let me update it, then you can update the PR.

((enjoy))
cr

**Comment by carlosfrodriguez** [comment]
Hi @urbalazs 

I updated the package to match OTRS 5.0.9 please update your pull request. BTW in some places we can move the creation of the Layout object a bit lower in the code just where is really needed.

((enjoy))
cr

**Comment by urbalazs** [comment]
Hi @carlosfrodriguez 
I update my PR to match OTRS 5.0.9. Please sync translatable strings with Transifex after merging.

**Comment by urbalazs** [comment]
Hi @carlosfrodriguez 
Can you check my updated PR?

**Comment by carlosfrodriguez** [comment]
Hi @urbalazs,

I will ASAP.

**Comment by urbalazs** [comment]
Hi @carlosfrodriguez 
Can you check my PR? It adds more translatable strings to language files.

**Comment by carlosfrodriguez** [comment]
Hi @urbalazs,

Thank you for your patience, this pull request was merged in https://github.com/OTRS/OTRSMasterSlave/commit/9b84e4ad3eb398525456571fe21c0edd375eee79 with very small modifications.

((enjoy))
cr

**Pull request closed without merging**